File: 프로그래머스/Level1/가운데 글자 가져오기/middle_word.java
class Solution {
    public String solution(String s) {
        String answer = "";
        int len_s = s.length();
        int locate = len_s/2;
        if (len_s % 2 ==0){
            answer += s.substring(locate-1,locate+1);
        }
        else{
            answer += s.substring(locate,locate + 1);
        }
        return answer;
    }
}

File: 프로그래머스/Level1/가운데 글자 가져오기/middle_word.py
def solution(s):
    answer = ''
    len_s = len(s)
    locte = len_s // 2
    if len_s % 2 ==0:
        answer += s[locte-1: locte+1]
    else:
        answer += s[locte : locte + 1]
    return answer
